hack/kind-up.sh

```diff
@@ -358,19 +357,19 @@ EOF
 fi
 
 # workaround https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files
-kubectl get nodes -o name |\
-  cut -d/ -f2 |\
-  xargs -I {} docker exec {} sh -c "sysctl fs.inotify.max_user_instances=8192"
-
-value=$(kubectl get cm -n kube-system kubeadm-config -oyaml | yq e '.data.ClusterConfiguration' | yq e '.apiServer.extraVolumes[0].hostPath')
-if [[ -n "$value" && "$value" != "null" ]]; then
-  kubectl get cm -n kube-system kubeadm-config -o yaml | \
-    sed -e "s#authorization-mode: RBAC,Node#authorization-mode: RBAC,Node,Webhook\n        authorization-webhook-config-file: $value#" | \
+for node in $(kubectl get nodes -o jsonpath='{.items[*].metadata.name}'); do
+  docker exec "$node" sh -c "sysctl fs.inotify.max_user_instances=8192"
+done
+
+authorization_webhook_config_file=$(kubectl -n kube-system get configmap kubeadm-config -o yaml | yq e '.data.ClusterConfiguration' | yq e '.apiServer.extraVolumes[0].hostPath')
```

Comment:
Do we need the `yq e`?

Reply:
Maybe a `yq -e` instead.

Reply:
What do you mean exactly? Eliminate which of the both `yq e`? 2 times of `yq e` is used because we first fetch the '.data.ClusterConfiguration' key from the ConfigMap. The result it is YAML again, hence we need to use `yq e` again  to fetch the  `.apiServer.extraVolumes[0].hostPath` field.

For the record, I didn't change this part. I only renamed the var to be something more meaningful than `value` and unified fetching the configmap (the namespace to be after kubectl, `configmap` to be used instead of the shorthand `cm`).

Reply:
> Maybe a yq -e instead.

`yq e` seems to be a shorthand for `yq eval`. From https://mikefarah.gitbook.io/yq#notice-for-v4.x-versions-prior-to-4.18.1:
> Since 4.18.1, yq's 'eval/e' command is the default command and no longers needs to be specified.

`yq -e` is:
>   -e, --exit-status                   set exit status if there are no matches or null or false is returned

I would rather drop the `e` part as we are using yq@v4.44.3 and it no longer needs to be specified since 4.18.1.